languages/python/sqlalchemy-oso/sqlalchemy_oso/partial.py

```diff
@@ -91,32 +91,32 @@ def translate_in(expression, session, model):
 
     # Partial In: LHS is an expression
     if isinstance(left, Expression):
-        path = dot_op_path(right)
+        path = dot_path(right)
         assert path
 
-        return translate_dot_op(
+        return translate_dot(
             path, session, model, functools.partial(emit_subexpression, left)
         )
     else:
         # Contains: LHS is not an expression.
         # TODO (dhatch) Missing check, left type must match type of the target?
-        path = dot_op_path(right)
+        path = dot_path(right)
         assert path
         path, field_name = path[:-1], path[-1]
-        return translate_dot_op(
+        return translate_dot(
             path, session, model, functools.partial(emit_contains, field_name, left)
         )
 
 
-def translate_dot_op(path: List[str], session: Session, model, func: EmitFunction):
+def translate_dot(path: Tuple[str], session: Session, model, func: EmitFunction):
```

Comment:
@dhatch as part of improving linting of the Python lib(s) we should look into setting up MyPy on CI (or removing these type annotations lest they drift out of sync).

Reply:
100% agree - i have this on my random list and haven't gotten to it